api/server/routes.go

```diff
@@ -71,12 +71,6 @@ func API(cfg Config, rti runtime.Initializer, execRouter *router.ExecRouter) {
 		})
 	})
 
-	r.Route("/ssh-keys/{owner}", func(r chi.Router) {
-		r.Post("/", SSHKeyAdd(rti))
-		r.Get("/", SSHKeyList(rti))
-		r.Post("/generate", SSHKeyGenerate(rti))
-	})
-
```

Comment:
Why were these removed?

Reply:
they are dead, they don't do anything

Reply:
Aren't they used by the CLI? https://github.com/unweave/cli/blob/master/cmd/keys.go#L85

Reply:
nope, what is used by the CLI is the handler that was in infra, i could have removed this from the master branch and it would be a pure negative diff

Reply:
we should be testing the self-hosted flows in this case